_data/simple-icons.json

```
@@ -1551,6 +1551,11 @@
             "hex": "334455",
             "source": "https://www.autohotkey.com"
         },
+        {
+            "title": "AutoIt Script",
+            "hex": "254768",
+            "source": "https://commons.wikimedia.org/wiki/File:AutoIt-logo.svg"
```

As the Wikimedia file is just the website logo, we should use the website as the `source`.
```suggestion
            "source": "https://www.autoitscript.com"
```

---

hmm, I used the Wikimedia as source and it clarifies copyright questions right on the spot, don't you think?

---

We only resort to Wikimedia as a `source` in the absence of any other official vector artwork, or raster artwork that can be converted to vector.

---

done